### PR TITLE
Modifies how open solver is called.

### LINKED ISSUE #1427: Adjusting call to open solver.
The way open solver is called changed in https://github.com/gnosis/dex-open-solver/pull/78, and it must be adjusted.

### PR DESCRIPTION
**Pull request by marcovc**
Fixes #1427 

Open solver invocation has changed gnosis/dex-open-solver#78. In particular the new packaging installs a script for the solver.
  
### Test Plan
How to test? I couldn't even run it here.

### CODE REVIEW

**Review by nlordell — COMMENTED** [review]
Maybe the open solver release version should be bumped in tandem in  [`.travis.yml`](https://github.com/gnosis/dex-services/blob/40aefb5404f5af5f223a4daee08079c1ddcbc505/.travis.yml#L12) build file.

**Comment by nlordell** [comment]
> How to test? I couldn't even run it here.

We test the open solver in Travis with our automated integration tests, so "Test Plan: CI" should be enough :smile: 

https://github.com/gnosis/dex-services/blob/40aefb5404f5af5f223a4daee08079c1ddcbc505/.travis.yml#L42-L45

**Review by nlordell — APPROVED** [review]

File: services-core/src/price_finding/price_finder_interface.rs [hunk 1]
```diff
@@ -99,10 +99,8 @@ pub fn execute_open_solver(
     input_file: &str,
     min_avg_fee_per_order: u128,
 ) -> Result<Output> {
-    let mut command = Command::new("python");
+    let mut command = Command::new("gp_match");
     let open_solver_command = command
-        .current_dir("/app/open_solver")
-        .args(&["-m", "src.match"])
```
Comment (bh2smith) [inline services-core/src/price_finding/price_finder_interface.rs:105]: This is great! So we no longer hard code `app/open_solver`! Amazing! is `gp_match` the name of the pip package?

**Comment by bh2smith** [comment]
This is the build failure:

```
manifest for gnosispm/dex-open-solver:v0.1.0 not found

The command "docker build --tag stablex-binary-public --build-arg SOLVER_BASE=gnosispm/dex-open-solver:$OPEN_SOLVER_VERSION -f driver/docker/rust/Dockerfile ." exited with 1.
```

Looks like the version bump hasn't been published.

File: services-core/src/price_finding/price_finder_interface.rs (hunk 1 above)
Comment (marcovc) [reply services-core/src/price_finding/price_finder_interface.rs:105]: It's the name of a script that gets installed system wide when the open solver is installed via pip.

**Review by bh2smith — APPROVED** [review]

**Comment by marcovc** [comment]
> Looks like the version bump hasn't been published.

**Comment by josojo** [comment]
You are soo close @marcovc 
I think the error is that your version tag in dex-open-solver is 0.1.0 instead of v0.1.0

**Review by josojo — APPROVED** [review]

**Pull request merged**